### PR TITLE
Edit travel entry

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/app/entries/entry/entry.component.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@use '../../../variables.scss' as *;
+
+.page-content {
+  position: relative;
+  z-index: 1;
+  background: map-get($sormas-accent, 50);
```
Comment (adriandumavitagroup) [inline src/app/entries/entry/entry.component.scss]: mat.get-color-from-palette

File: src/app/entries/entry/entry.component.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.tab-nav-header {
+  position: sticky;
+  z-index: 3;
+  top: 164px;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 25px;
+  border-bottom: solid 2px map-get($sormas-accent, 200);
```
Comment (adriandumavitagroup) [inline src/app/entries/entry/entry.component.scss]: mat.get-color-from-palette

File: src/app/entries/entry/entry.component.scss (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/entries/entry/entry.component.scss]: Fixed.

File: src/app/entries/entry/entry.component.scss (hunk 2 above)
Comment (sandorbanvitagroup) [reply src/app/entries/entry/entry.component.scss]: Resolved.

File: src/app/entries/entry/entry.component.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        (completion)="onEntryDelete()"
+        class="actions"
+      ></app-entity-actions>
+    </div>
+  </header>
+
+  <app-info-bar *ngIf="showTabs" [data]="entry" [type]="'ENTRY'"></app-info-bar>
+
+  <div class="tab-nav-header">
+    <nav mat-tab-nav-bar *ngIf="true">
```
Comment (danielrus) [inline src/app/entries/entry/entry.component.html]: this is always shown

File: src/app/entries/entry/entry.component.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    // eslint-disable-next-line no-console
+    console.log('add entry');
+  }
+
+  ngOnInit(): void {
+    const routeParams = this.activeRoute.snapshot.params;
+    this.currentSubPage = this.helperService.getCurrentSubpage(this.router.url, entryLinks);
+    this.entryId = routeParams.entryId;
+    this.links = entryLinks(this.entryId);
+    this.entryService.getById(this.entryId).subscribe({
```
Comment (danielrus) [inline src/app/entries/entry/entry.component.ts]: unsubscribe this on destroy

File: src/app/entries/entry/entry.component.html (hunk 3 above)
Comment (sandorbanvitagroup) [reply src/app/entries/entry/entry.component.html]: Resolved.

File: src/app/entries/entry/entry.component.ts (hunk 4 above)
Comment (sandorbanvitagroup) [reply src/app/entries/entry/entry.component.ts]: Resolved

**Review by danielrus — APPROVED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**